source/usage-examples/find.txt

```diff
@@ -0,0 +1,164 @@
+
+``find`` returns the following documents for such a query:
+
+.. code-block:: javascript
+
+   const cursor = collection.find(query);
+
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 }
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 }
+
+Sort Order
+~~~~~~~~~~
+
+Use sort to change the order in which ``find`` returns documents. Pass
+the following sort order to ``find`` to ensure that the operation
+returns fruit documents with higher ratings before fruit documents with
+lower ratings:
+
+.. code-block:: javascript
+
+   const sort = { rating: -1 }
+
+In this case, the number ``-1`` tells ``find`` to sort the fruit in
+descending order by rating, so the highest rated fruit comes first and
+the lowest rated fruit comes last. ``find`` returns the following
+documents when this sort is used with no query:
+
+.. code-block:: javascript
+
+   options = {};
+   options.sort = sort;
+   const cursor = collection.find({}, options);
+
+   { "_id": 4, "name": "avocados", "qty": 3, "rating": 5 }
+   { "_id": 1, "name": "apples", "qty": 5, "rating": 3 }
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 }
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 }
+
+
+Rewriting this sort to use a value of
+``1`` instead of a ``-1``, will sort our fruit in *ascending* order
+by rating, so the lowest rated fruit would come first. If you do not
+specify a sort order, the order of documents returned is
+`not guaranteed <https://docs.mongodb.com/manual/reference/method/cursor.sort/#result-ordering>`_.
+
+Projection
+~~~~~~~~~~
+
+Use a projection to control which fields ``find`` includes in the
+returned documents. Pass the following projection to only return the
+names of the fruits:
+
+.. code-block:: javascript
+
+   const projection = { name: 1 }
+
+``name`` has a value of ``1`` here to indicate that ``find`` should
+*include* the ``name`` field of each document returned. This implicitly
+*excludes* any fields that aren't explicitly included, so remember to
+specify any fields you want ``find`` to return with a ``1``. Passing
+this projection to ``find`` with no query and no sort order yields the
+following results:
+
+.. code-block:: javascript
+
+   options = {};
+   options.projection = projection;
+   const cursor = collection.find({}, options);
+
+   { "_id": 1, "name": "apples" }
+   { "_id": 2, "name": "bananas" }
+   { "_id": 3, "name": "oranges" }
+   { "_id": 4, "name": "avocados" }
+
+Despite the fact that this query only explicitly included the ``name``
+field, the ``_id`` field is returned as well! This happens because the
+``_id`` field is a unique identifier for each document, and if you
+want to refer to these documents again, you'd create a query document
+matching by ``_id`` in case there are duplicate ``name`` values in your
+collection. MongoDB anticipates that you might want a unique identifier
+for any documents you've found, so ``_id`` is always included unless you
+explicitly exclude it.
```

Comment:
This paragraph is a little confusing. I'd emphasize first that `_id` is a special case and then give context as to why.

Reply:
i.e. this happens because "projections treat the `_id` field differently than other fields", not just because it's a unique ID.

Reply:
Fair, I was trying to do the opposite: point out why _id is especially useful, then mention that mongodb treats it differently as a result. Since it sounds like mine can be a bit confusing, I'll change mine to the opposite style like you suggest to make it clearer.